lib/absinthe/blueprint/schema.ex

```diff
@@ -141,6 +141,11 @@ defmodule Absinthe.Blueprint.Schema do
     build_types(rest, [enum | stack], buff)
   end
 
+  defp build_types([%Schema.InputValueDefinition{} = arg, {:desc, desc} | rest], stack, buff) do
```

Comment:
What's weird is that this shouldn't be necessary because the https://github.com/absinthe-graphql/absinthe/blob/ad2e4c2508379c323ed37b1556e224865f350fba/lib/absinthe/blueprint/schema.ex#L110 clause should already handle this. Doing a lookahead to find the desc should be exactly identical to just moving one more item through the stack, matching on desc, and modifying the previous thing.

However if I remove this clause locally, the tests no longer pass. Investigating.

Reply:
Ah of course, this is what you were saying, I see it now. https://github.com/absinthe-graphql/absinthe/blob/ad2e4c2508379c323ed37b1556e224865f350fba/lib/absinthe/blueprint/schema.ex#L149 eagerly removes the InputValue definition from the stack, so further items can't do anything to it.